indexer/processor.js

```diff
@@ -950,9 +907,6 @@ class NinaProcessor {
     }
       for await (let newPost of newPosts) {
         try {
-          if (blacklist.includes(newPost.publicKey.toBase58())) {
```

Comment:
we have restricted posts - the blacklist was a combination of both - not sure which ones were posts though - this was likely to handle posts that were double published - maybe we just let it run and see which ones come up twice and think of solution (prob adding post type to restricted.type).  shouldnt be  too much of an issue if an old post is doubled

Reply:
we can probably leverage the type for post and if we notice the double we can just add it to the restricted table on an as needed basis.